src/framework/renderer/js/fluidRenderer.js

```diff
@@ -11,10 +11,9 @@ You may obtain a copy of the ECL 2.0 License and BSD License at
 https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
 */
 
-(function ($, fluid) {
-    "use strict";
-
+"use strict";
 
+(function ($) {
```

Comment:
Is this still needed for jQuery? It is removed from other files.

Reply:
I left the closure in since there are a lot of local variables that would otherwise leak out into the global scope, and it is not worth reengineering this file much since it will be deleted in the next release. I could remove the jQuery argument from the closure however.

Reply:
I wonder if we should remove jQuery. On one hand it probably doesn't matter, on the other, it might be confusing to and make someone in the future question why it's there.

Reply:
Simplified this and ImageReordererTests closures